lua/deadcolumn/configs.lua

```diff
@@ -1,9 +1,19 @@
 local M = {}
 
+local function displaywidth(line)
+  if line:find('%c') then
```

Comment:
I think it might be better to use:
```suggestion
  if vim.fn.type(line) == 10 then
```